### PR TITLE
Page 404 resolving conflicts

### PR DESCRIPTION
**Pull request by LUISEDOCCOR**
# Page 404 🚀
### This pr is thanks to @juanpablo-is and @Zyruks 🙌

https://private-user-images.githubusercontent.com/14036522/320619346-da9136f9-97d6-4629-bc01-c235a56351da.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI2MDM4OTgsIm5iZiI6MTcxMjYwMzU5OCwicGF0aCI6Ii8xNDAzNjUyMi8zMjA2MTkzNDYtZGE5MTM2ZjktOTdkNi00NjI5LWJjMDEtYzIzNWE1NjM1MWRhLm1wND9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA4VDE5MTMxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU0YTgwMGZhZDUwOGFiNDZkMmExMWIyNzc5YjMwODNmYzc0ZGRhNTYzZjg2OWRmOGE1OGM3YzVlOGYwMjBkN2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LqZ_3As5qMs0JWj640fBxPFxqgyyYLI4ShjGDOIDL00

The hamster **doesn't believe it**, I take it from https://codepen.io/jkantner/pen/wvqeXrQ

image

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -34,6 +34,13 @@
   );
 
   /* Carousel config end */
+
+  /* Hamster config start */
+  
+  --bg: hsl(90,90%,20%);
+	--fg: hsl(223,10%,10%);
```
Comment (Zyruks) [inline src/index.css:41]:
```suggestion
  --bg: hsl(90,90%,20%);
  --fg: hsl(223,10%,10%);
```

Fix indentation.

File: src/index.css [hunk 2]
```diff
@@ -367,3 +374,199 @@ body {
     stroke-width: 1;
   }
 }
+
+/************************************************/
+/*                 HAMSTRR CLASSES           */
```
Comment (Zyruks) [inline src/index.css:379]:
```suggestion
/*                 HAMSTER CLASSES           */
```

Misspelled

File: src/pages/NotFound/NotFound.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                <div className="hamster__limb hamster__limb--fl"></div>
+                <div className="hamster__limb hamster__limb--br"></div>
+                <div className="hamster__limb hamster__limb--bl"></div>
+                <div className="hamster__tail"></div>
+              </div>
+            </div>
+            <div className="spoke"></div>
+          </div>
+        </article>
+      </main>
```
Comment (Zyruks) [inline src/pages/NotFound/NotFound.tsx:52]:
```suggestion
      <main className="relative w-full z-10 text-cWhite font-dmsans">
        <section className="container mx-auto px-5 max-w-7xl mx-auto grid md:grid-cols-[1fr_.4fr] place-items-center place-content-center h-[100svh]">
          <article className="grid gap-4 text-shadow-md px-5">
            <h1 className="text-7xl font-semibold">Error 404</h1>
            <h2 className="text-4xl font-semibold">PÁGINA NO ENCONTRADA</h2>
            <p className="text-3xl">
              ¡Hola! Lo sentimos, pero no pudimos encontrar lo que buscabas. Verifica que la dirección URL sea correcta
            </p>
            <Link to={ROUTE.home}>
              <Button size={ButtonSize.xl} variant={Variant.ghost} onClick={() => {}} className="flex gap-2 items-center">
                <span>IR A INICIO</span>
                <div className="i-lucide:arrow-up-right  w-8 h-8 bg-gradient-to-rb from-primary-600 to-secondary-500" />
              </Button>
            </Link>
          </article>

          <article className="hidden md:block">
            {/* https://codepen.io/jkantner/pen/wvqeXrQ */}
            <div className="wheel-and-hamster" role="img" aria-label="Orange and tan hamster running in a metal wheel">
              <div className="wheel"></div>
              <div className="hamster">
                <div className="hamster__body">
                  <div className="hamster__head">
                    <div className="hamster__ear"></div>
                    <div className="hamster__eye"></div>
                    <div className="hamster__nose"></div>
                  </div>
                  <div className="hamster__limb hamster__limb--fr"></div>
                  <div className="hamster__limb hamster__limb--fl"></div>
                  <div className="hamster__limb hamster__limb--br"></div>
                  <div className="hamster__limb hamster__limb--bl"></div>
                  <div className="hamster__tail"></div>
                </div>
              </div>
              <div className="spoke"></div>
            </div>
          </article>
        </section>
      </main>
      
      Better html structure, and fix a problem with the hamster animation
      </main>
```



https://github.com/Afordin/hackafor-2/assets/14036522/da9136f9-97d6-4629-bc01-c235a56351da

**Review by Zyruks — CHANGES_REQUESTED** [review]
Good PR, Love the Hamster animation, great Job.

Check the footer z-index too, cuz the Background animation it's covering it.

**Comment by LUISEDOCCOR** [comment]
Thank you very much @Zyruks 🙌

**Pull request merged**